modules/programs/zsh.nix

```diff
@@ -623,8 +656,8 @@ in
         # Aliases
         ${aliasesStr}
         ''
-      ] 
-      ++ (mapAttrsToList (k: v: "alias -g ${k}=${lib.escapeShellArg v}") cfg.shellGlobalAliases) 
+      ]
+      ++ (mapAttrsToList (k: v: "alias -g ${k}=${lib.escapeShellArg v}") cfg.shellGlobalAliases)
```

Comment:
Seems like this was just `./format` changes?  
I think it's best not to include those in your changes to keep the blame useful.

Reply:
Yes, this was the result of ./format. Oops! Will fix soon.